src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -244,7 +244,7 @@
         <div class="{% if form is not rootform %}invalid-feedback d-block{% else %}alert alert-danger{% endif %}">
             <ul class="list-unstyled mb-0">
                 {%- for error in errors -%}
-                    <li>{{ error.message }}</li>
+                    <li><span class="initialism form-error-icon badge badge-danger">{{ 'error'|trans }}</span> <span class="form-error-message">{{ error.message }}</span></li>
```

Comment:
just a raw idea: instead of "error", we put that "X" unicode cross,
then we add a translation table where this X maps to "ERROR" in english, same in a few other languages,
but if we don't have the translation for the word "error", the cross is displayed instead - as it is more universal than "error", isn't it?
makes sense?